From df809fd0b505b61d718fe1dc7fe19a79f9336cd9 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Wed, 24 May 2023 20:21:33 +0200
Subject: [PATCH] Use pkgconfig in order to find libproxy configuration

>=libproxy-0.5 moved proxy.h into a non-default include search path.

See also:
https://github.com/libproxy/libproxy/issues/226#issuecomment-1557064225
https://bugs.gentoo.org/906879

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 src/network/configure.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/network/configure.json b/src/network/configure.json
index 271ff164ac..ffba2d1eea 100644
--- a/src/network/configure.json
+++ b/src/network/configure.json
@@ -53,7 +53,7 @@
             },
             "headers": "proxy.h",
             "sources": [
-                "-lproxy"
+                { "type": "pkgConfig", "args": "libproxy-1.0" }
             ]
         },
         "openssl_headers": {
-- 
2.40.1

